src/pages/fareConfirmation.tsx

```diff
@@ -52,6 +63,7 @@ export const buildFareConfirmationElements = (
             href: 'definePassengerType',
         });
     }
+
     if (passengerType.proof && passengerType.proofDocuments) {
```

Comment:
This is an interesting one, because as written we are allowing the user to go back and change their passenger type from school pupil to something else, which I don't think is intended.